polaris-icons/icons/FilterMinor.svg

```diff
@@ -0,0 +1,3 @@
+<svg viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M2 6C2 5.44772 2.44772 5 3 5H17C17.5523 5 18 5.44772 18 6C18 6.55228 17.5523 7 17 7H3C2.44772 7 2 6.55228 2 6ZM5 10C5 9.44772 5.44772 9 6 9H14C14.5523 9 15 9.44772 15 10C15 10.5523 14.5523 11 14 11H6C5.44772 11 5 10.5523 5 10ZM8 13C7.44772 13 7 13.4477 7 14C7 14.5523 7.44772 15 8 15H12C12.5523 15 13 14.5523 13 14C13 13.4477 12.5523 13 12 13H8Z" fill="#5C5F62"/>
```

Comment:
You should be able to remove the following attributes and then the CI checks should pass
```suggestion
<path d="M2 6C2 5.44772 2.44772 5 3 5H17C17.5523 5 18 5.44772 18 6C18 6.55228 17.5523 7 17 7H3C2.44772 7 2 6.55228 2 6ZM5 10C5 9.44772 5.44772 9 6 9H14C14.5523 9 15 9.44772 15 10C15 10.5523 14.5523 11 14 11H6C5.44772 11 5 10.5523 5 10ZM8 13C7.44772 13 7 13.4477 7 14C7 14.5523 7.44772 15 8 15H12C12.5523 15 13 14.5523 13 14C13 13.4477 12.5523 13 12 13H8Z" fill="#5C5F62"/>
```

Also this svg should just be a one-liner. Otherwise this looks good to me. Let me know when you've updated or if you want a hand

Reply:
@kyledurand ok just made these changes. should be good to go now!